src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] color = Color.values();
+
+    public static String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
```

Comment:
You've already stored the enum values in the `color` array. You should use it here instead of calling `Color.values()` again to avoid unnecessary calls to `values()` which creates a new array each time.